### ISSUE TITLE
Perimeter seam can be broken with Corners and nearest seam placement

### ISSUE THREAD

**Issue by neophyl** [supermerill/SuperSlicer#205]
### Version
2.2.49

### Operating system type + version
Win 10 64 bit

### 3D printer brand / version + firmware version (if known)
N/A

### Behaviour
When using Corners seam placement on this (nearly) single wall tube the perimeters are broken.  The effect can also be seen on Nearest too but only for several layers (it doesn't travel all the way up, just for about 24 layers).

![corner seam](https://user-images.githubusercontent.com/46955527/79984058-8b9a4b00-84a0-11ea-972b-6bc03d5ab785.png)

Changing to a single perimeter stops the effect.  However for most models multiple perimeters are desirable.
Adding a second instance of the model stops the broken perimeters too.

Rear/Aligned/Random placement does not exhibit the behaviour.

_Is this a new feature request?_
No

#### Project File (.3MF) where problem occurs
[Corners_seam.zip](https://github.com/supermerill/Slic3r/files/4516171/Corners_seam.zip)

**Comment by liftbag** [comment]
As workaround, you can drop the external perimeter width to 0.42mm and enable overlapping perimeters.

<img width="1711" alt="Schermata 2020-04-23 alle 12 56 52" src="https://user-images.githubusercontent.com/14921654/80091866-2149df00-8562-11ea-95d9-3a06955988ac.png">

**Comment by supermerill** [comment]
neophyl  said that a Rear/Aligned/Random seam works.

**Comment by liftbag** [comment]
> neophyl said that a Rear/Aligned/Random seam works.
> overlapping perimeters will create a huge over-extrusion.

Yes, only nearest and corners and nearest produce artifacts.
I did some tests to understand what triggered the problem, and I found that reducing the extrusion width, along with enabling overlapping perimeters, the artifacts disappeared, even with nearest and corners seam. But using the two profile changes individually, it doesn't resolve. It actually looks like a bug.
